app/views/users/show.html.erb

```diff
@@ -1 +1,10 @@
-<h1> User Page</h1>
\ No newline at end of file
+<h1> User Page</h1>
+<%= render partial: "shared/usercard", locals: { user: @user} %>
+<div>
+  <div>Bio</div>
+  <div>
+    <%= @user.bio %>
+</div>
+<% @posts.each do |post|%>
+  <%= render partial: "shared/postcard", locals: { post: post} %>
+<% end %>
```

Comment:
- [ ] Please edit the user show view design to be like the given design and kindly ensure that you add all elements in the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/180583100-4a7dcfdc-1d78-414f-b7de-9a14741721b7.png)  | ![image](https://user-images.githubusercontent.com/68796232/180583111-ffdbc7bf-618a-434e-91c0-c41c0942ac74.png) |